test/corpus/literals.txt

```diff
@@ -286,7 +286,7 @@ strings
 "_"
 "# not a comment"
 "🔮"
-" 	
```

Comment:
Can you preserve this whitespace?

Reply:
yes, my editor keeps auto-removing it due to removing trailing whitespace